tests/Integration/config/5.4/messenger.yaml

```diff
@@ -1,5 +1,6 @@
 framework:
     messenger:
+        reset_on_message: true
```

Comment:
Also to fix in tests:
```
Remaining indirect deprecation notices (1)
  1x: Since symfony/framework-bundle 5.4: Not setting the "framework.messenger.reset_on_message" configuration option is deprecated, it will default to "true" in version 6.0.
    1x in CliTest::testDumpSitemapUsingCLI from Presta\SitemapBundle\Tests\Integration\Tests
```

Reply:
Nice catch